index.html

```diff
@@ -1,38 +1,298 @@
+      </div>
+      <nav class="header2">
+        <h3>Welcome</h3>
+        <img class="menu-button" src="./images/Enabled.png" alt="mobile-menu" />
+      </nav>
     </header>
     <section class="heading">
-        <div></div>
-        <h1>Hey there, I am Chukwuemeka <br><span>I am a Software Developer</span></h1>
-        <p class="heading-paragraph"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-            do not hestiate to contact me.</p>
-        <ul class="follow-link">
-            <li><a href="https://twitter.com"><img src="./images/twitter.svg" alt=""></a></li>
-            <li><a href="https://linkedin.com"><img src="./images/Linkedin icon.svg" alt=""></a></li>
-            <li><a href="https://medium.com"><img src="./images/medium.svg" alt=""></a></li>
-            <li><a href="https://github.com"><img src="./images/github.svg" alt=""></a></li>
-            <li><a href="https://angellist.com"><img src="./images/angellist.svg" alt=""></a></li>
-        </ul>
+      <div></div>
+      <h1>
+        Hey there, I am Chukwuemeka <br /><span>I am a Software Developer</span>
+      </h1>
+      <p class="heading-paragraph">
+        I can help you build a product, feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, do not hestiate to contact me.
+      </p>
+      <ul class="follow-link">
+        <li>
+          <a href="https://twitter.com"
+            ><img src="./images/twitter.svg" alt=""
+          /></a>
+        </li>
+        <li>
+          <a href="https://linkedin.com"
+            ><img src="./images/Linkedin icon.svg" alt=""
+          /></a>
+        </li>
+        <li>
+          <a href="https://medium.com"
+            ><img src="./images/medium.svg" alt=""
+          /></a>
+        </li>
+        <li>
+          <a href="https://github.com"
+            ><img src="./images/github.svg" alt=""
+          /></a>
+        </li>
+        <li>
+          <a href="https://angellist.com"
+            ><img src="./images/angellist.svg" alt=""
+          /></a>
+        </li>
+      </ul>
     </section>
-</body>
-</html>
\ No newline at end of file
+    <main>
+      <h1>My Recent Works</h1>
+      <section class="grid-display">
+        <div class="card">
+          <div class="card-body">
+            <img
+              src="./images/photo-1533035353720-f1c6a75cd8ab.jpg"
+              alt=""
+              class="card-image"
+            />
+            <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+            <div class="card-tags">
+              <p>Ruby on Rails</p>
+              <p>css</p>
+              <p>JavaScript</p>
+              <p>html</p>
+            </div>
+            <button class="card-button">See Project</button>
+          </div>
+        </div>
```

Comment:
- [ ] As mentioned in the project requirements **_"Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card."_**   please use flex for layout of each individual card, we need you to be flex ninja :ninja:

Reply:
Corrected. Thank you for your review